sql_to_code/parsers/alter_table/parser.py

```diff
@@ -1,32 +1,33 @@
-import re
+from pyparsing import *
 
 from .models import ForeignKey, Reference
 
-source_table_name_regex = re.compile('ALTER TABLE "(?P<table_name>\w+)"')
-foreign_key_name_regex = re.compile('ADD FOREIGN KEY \("(?P<foreign_key>\w+)"\)')
-result_table_name_regex = re.compile('REFERENCES "(?P<result_table_name>\w+)"')
-result_table_field_name_regex = re.compile(
-    'REFERENCES "\w+" \("(?P<result_table_field_name>\w+)"\);'
+table_name_schema = CaselessKeyword("alter table") + QuotedString('"')("table_name")
```

Comment:
So, is it expected than `table_name` should be quoted in a way "NAME"? Is my assumption correct that it won't work with single quotes? If I understand it correctly from [their documentation](https://pythonhosted.org/pyparsing/pyparsing-pysrc.html#QuotedString), you can use multiple symbols in order to decide which quotes could be used. What do you think?

Reply:
Also it could be unquoted at all. Guess this case could also be handled here